libscript/include/libscript/script.h

```diff
@@ -27,6 +27,8 @@
 #include <foundation-system.h>
 #endif
 
+#include <ffi.h>
```

Comment:
This causes a build error on windows - I suggest not having the include here (as libffi should be private to libscript), and replace ffi_abi with its typedef'd type :)

Reply:
hmm... ok, I won't need the patch to thirdparty then which resolved the build error